mid_assessment_A/1_factorial_pg.rb

```diff
@@ -0,0 +1,21 @@
+# Write a program to calculate the factorial of a given number.
+
+# `Way-1` to calculate factorial
+def fact(n)
```

Comment:
avoid using single letter variable names from now. Always use meaningful and precise names